src/data/general-cs-quiz.ts

```diff
@@ -1319,6 +1319,15 @@ const generalCSQuiz = [
     Explanation:
       "The worst case time complexity for linear search O(n) because we have to iterate through n elements. The worst case time complexity for binary search is O(logn) because it is a divide and conquer algorithm.",
     Link: "https://www.freecodecamp.org/news/search-algorithms-linear-and-binary-search-explained/"
+  },
+  {
+    Question: "What is backbone of Merge Sort?",
+    Answer: "Recursion",
+    Distractor1: "An Array",
+    Distractor2: "A Data Structure",
+    Distractor3: "Complex Alghorithms",
+    Explanation: "The Merge Sort algorithm works by breaking down an unsorted list into two halves, for this it uses Recursion.",
```

Comment:
```suggestion
    Explanation: "The Merge Sort algorithm works by recursively breaking down an unsorted list into two halves until it reaches a point where it can no longer be broken down.",
```